apisix/plugin.lua

```diff
@@ -297,6 +297,7 @@ local function merge_service_route(service_conf, route_conf)
     local new_conf = core.table.deepcopy(service_conf)
     new_conf.value.service_id = new_conf.value.id
     new_conf.value.id = route_conf.value.id
+    new_conf.modifiedIndex = route_conf.modifiedIndex
```

Comment:
`new_conf.modifiedIndex` -> new_conf.modified_index

Reply:
the "modifiedIndex" while be used in other space as route modified Index, if I change this key ,should change in all place.

Reply:
got it